honeypot/Datadog.Dependabot.Honeypot.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
@zacharycmontoya 
I really miss docs describing how to use it and in general all the tools you have built for managing vendor dependencies.